spec/models/entity_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe Entity, type: :model do
+  subject { Entity.new(name: 'LendMoney', amount: 15) }
+  before { subject.save }
+
+  it 'name should be "LendMoney".' do
+    expect(subject.name).to eq 'LendMoney'
```

Comment:
Some of your tests failed. Kindly look into this.

![image](https://user-images.githubusercontent.com/10332499/164579586-66520196-10a0-4cc1-ad9b-f2d8ccec157a.png)

Reply:
Hi @micheaol, thanks for your kind feedback. Although I already state some instructions in the README file about how to execute the tests in commit dcfe9ec, I made some improvements in order to better notice this instruction on commit 307faf0.